Doc/library/inspect.rst

```diff
@@ -1405,6 +1405,7 @@ is considered deprecated and may be removed in the future.
           frame = inspect.currentframe()
           try:
               # do something with the frame
+              ...
```

Comment:
```suggestion
              ...  # do something with the frame
```

I propose to use this style where possible: we don't need `...` as a separate expressions, with a comment to looks more readable in my opinion.

Reply:
I agree, I'll rework these.